docs/content/docs/data-structures/sequences.md

```diff
@@ -255,7 +255,7 @@ type of content they support. SharedNumberSequence only supports numbers as cont
 any JSON serializable object. Both DDSes support inserting, removing, and annotating content. Each item -- that is, each
 number or object -- will occupy a single position in the sequence.
 
-An important note is that, unlike an array, positions are not guaranteed remain constant. The position of an item can
+An important note is that, unlike an array, positions are not guaranteed to remain constant. The position of an item can
```

Comment:
Can you make this change in the source file instead? See the comment at the top of the file:

>Don't manually edit the auto-generated content below. Instead, edit the source file (look at the path argument) and then run "npm run build:md-markdown" from the docs folder. That will update all autogenerated content, so you should commit the resulting changes.

Reply:
It's probably easier in this case to just add the missing word to the source file, instead of running `build:md-markdown`.

Reply:
@tylerbutler I've implemented the changes you suggested! I did end up changing the source file and generated file (this file) separately/directly without running `build:md-markdown`